apps/docs/components/cardLink/cardLink.css

```diff
@@ -0,0 +1,126 @@
+@property --angle {
+    syntax: "<angle>";
+    initial-value: -45deg;
+    inherits: false;
+}
+
+.hd-cardlink {
+    --card-color: #332c7c;
+    --card-color-translucent: #c58d9baa;
+
+    backdrop-filter: blur(2rem);
+    background-color: var(--card-color);
+    border-radius: var(--hd-border-radius-lg);
+    color: var(--hd-white);
+    display: flex;
+    flex-direction: column;
+    flex: 1 1 auto;
+    height: 14rem;
```

Comment:
very small detail, but I'm thinking maybe this should use min-height instead of a fixed height.
Reason being, text could possibly get cut off as you resize the page or if the text gets adjusted later on.